_includes/package_readme.html

```diff
@@ -0,0 +1,12 @@
+{% capture readme %}
+    {% include babel/packages/{{ page.package }}/README.md %}
+{% endcapture %}
+
+{{ readme
+    | newline_to_br
+    | split: "<br />"
+    | slice: 4, 1000
```

Comment:
This is to remove the first 4 header lines at the top of every readme https://raw.githubusercontent.com/babel/babel/master/packages/babel-plugin-syntax-async-generators/README.md - they all ahve it so it should be ok as a workaround (using shift).